server/app/controllers/TaskSolver.scala

```diff
@@ -67,5 +100,9 @@ object TaskSolver {
 
   def sbt(command: String): Try[Boolean] = Try(Seq("sbt", command).! == 0)
 
-  lazy val sbtInstalled = sbt("sbtVersion").isSuccess // no exception, so sbt is in the PATH
+  // no exception, so sbt is in the PATH
+  lazy val sbtInstalled = sbt("--version").isSuccess
```

Comment:
I have sbt version 0.13.9, maybe this key "--version" was removed in 13.7. Not sure